tests/legacy/config.js

```diff
@@ -48,6 +56,7 @@ module.exports = function(OfflinePluginOptions, testFlags) {
 
   if (webpackMajorVersion === '4') {
```

Comment:
should probably use `if (parseInt(webpackMajorVersion, 10) >= 4)` to be future proof.